### PR TITLE
Remove binary strip

### PR DESCRIPTION
**Pull request by fabienrohrer**
Implement proposition #119

### CODE REVIEW

**Comment by fabienrohrer** [comment]
test me

**Comment by fabienrohrer** [comment]
👍

File: src/webots/Makefile [hunk 1]
```diff
@@ -180,11 +180,9 @@ endif
...
 ifeq ($(MAKECMDGOALS),release)
 CFLAGS += -O3 -DNDEBUG
-ifeq ($(OSTYPE),darwin)
-STRIP = strip $(TARGET)
-else
-LD_FLAGS += -s
-endif
+ ifneq ($(OSTYPE),darwin)
+  LD_FLAGS += -s
+ endif
```
Comment (omichel) [inline src/webots/Makefile]: We usually don't indent if/endif in Makefile

Comment (fabienrohrer) [reply src/webots/Makefile]: Ok, I just homogenized the indentation, by dropping it in the entire file.

**Comment by fabienrohrer** [comment]
test me

**Pull request merged**